.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,70 @@
+# GitHub issue form. For more information see:
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+
+name: Report something is not working properly 🐛
+description:
+  Use this if there is something that is not working properly.  If you are not
+  sure or you need help making something work, please ask a question instead.
+labels: priority:❓, type:bug
+body:
+  - type: markdown
+    attributes:
+      value:
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please tell us what happen that it shouldn't have.
+      placeholder: What happen that it shouldn't have.
+    validations:
+      required: true
+  - type: textarea
+    id: what-expected
+    attributes:
+      label: What did you expected instead?
+      description: Please tell us what did you expect to happen.
+      placeholder: What did you expect to happen.
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Affected version(s)
+      description:
+        Which version(s) have you tried and have this issue? (if you see this
+        issue **not** happening on any version, please let us know in the
+        "Extra information")
+      multiple: true
+      options:
+        - I don't know (version:❓)
+        - An older version (version:older)
+        - v0.10.x (version:0.10.x)
+    validations:
+      required: true
+  - type: dropdown
+    id: part
+    attributes:
+      label: Affected part(s)
+      description:
+        Which parts of the repo are affected by this issue? Select all that
+        applies
+      multiple: true
+      options:
+        - I don't know (part:❓)
+        - Documentation (part:docs)
+        - Unit, integration and performance tests (part:tests)
+        - Build script, CI, dependencies, etc. (part:tooling)
+        - Protocol buffer definition (part:protobuf)
+        - Python bindings (part:python)
+    validations:
+      required: true
+  - type: textarea
+    id: extra
+    attributes:
+      label: Extra information
+      description:
+        Please write here any extra information you think it might be relevant,
+        like if this didn't happened before or if you suspect where the problem
+        might be
```

Comment:
A slight change recommended here:
` .. you think might be relevant, e.g., if this didn't happen before, or ..`